### DISCUSSION TITLE
Among Us Online

### DISCUSSION THREAD

**Discussion by giscus** [astrogames06/astrogames06.github.io#17] [Q&A]
# Among Us Online

https://astrogames06.github.io/game.html

**Comment by ur-mom847** [comment]
bruh theres never anyone online

**Comment by Kell3n** [comment]
Then wait for someone to be on, DUH 🙄

**Comment by 1amamazing2468** [comment]
I actually love this game!

**Reply by astrogames06** [reply]
thanks

**Comment by ruthie550** [comment]
this game is low key so fun.
I wish I could type what I want in the chat

**Comment by braedenwoolen11-lab** [comment]
Hey astro, can you add chat to where we can type what we want?

**Reply by ruthie550** [reply]
yea can u?

**Reply by astrogames06** [reply]
Do you mean a game request form? May you please specify.

**Reply by ruthie550** [reply]
instead of giving us stuff to use in the chat make it a typing space so we can type what we want.

**Reply by braedenwoolen11-lab** [reply]
yea that what I was talking about. Sorry I wasn't specific astro

**Reply by astrogames06** [reply]
> yea that what I was talking about. Sorry I wasn't specific astro

i didnt make the game so i cant really change what happens within the actual. but i can maybe make a live chat for each game within the website. is that u want?

**Reply by braedenwoolen11-lab** [reply]
yes please!

**Comment by braedenwoolen11-lab** [comment]
how do you change ur username in the game? is it just bots that have those names?

**Comment by braedenwoolen11-lab** [comment]
how do people have ice cube hats?????

**Reply by watorbotl** [reply]
type "chilly" on ur keyboard
